mmv1/third_party/terraform/website/docs/r/container_cluster.html.markdown

```diff
@@ -891,6 +891,26 @@ linux_node_config {
 
 * `node_group` - (Optional) Setting this field will assign instances of this pool to run on the specified node group. This is useful for running workloads on [sole tenant nodes](https://cloud.google.com/compute/docs/nodes/sole-tenant-nodes).
 
+* `sole_tenant_config` (Optional)  Allows specifying multiple [node affinities](https://cloud.google.com/compute/docs/nodes/sole-tenant-nodes#node_affinity_and_anti-affinity) useful for running workloads on [sole tenant nodes](https://cloud.google.com/kubernetes-engine/docs/how-to/sole-tenancy). `node_affinity` structure is [documented below](#nested_node_affinity).
+
+```bcl
+sole_tenant_config {
+  node_affinity {
+    key = "compute.googleapis.com/node-group-name"
+    operator = "IN"
+    values = ["node-group-name"]
+  }
+}
+```
+
+<a name="nested_node_affinity"></a>The `node_affinity` block supports:
```

Comment:
Ah, missed this the first time around. Do you mind putting this under `advanced_machine_features`? At line 916 in the current display. Right now it displays that as if it were a subfield of `node_affinity`.

Reply:
Good catch. I added the change but the PR isn't picking up the push even though it shows up in the branch https://github.com/jasonkramberger/magic-modules/tree/sole-tenant-config

Reply:
Whoops, replied on main instead of here: https://github.com/GoogleCloudPlatform/magic-modules/pull/8072#issuecomment-1581255187